koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -514,6 +518,249 @@ GROUP BY aws.lineitem_usagestartdate,
     aws.costcategory,
     17, -- tags
     aws.matched_tag
+{% endif %}
+;
```

Comment:
Should this be inside the template conditional so it is omitted when `unattributed_storage` is `False`?

Reply:
Based on what is in https://github.com/project-koku/koku/blob/main/docs/jinja_templating.md I think the `;` needs to be after the endif

Reply:
That's an interesting nuance.